src/AutoMapper/ApiCompatBaseline.txt

```diff
@@ -1 +1,6 @@
-Total Issues: 0
+Compat issues with assembly AutoMapper:
+InterfacesShouldHaveSameMembers : Interface member 'public void AutoMapper.IConfigurationProvider.Validate(AutoMapper.ValidationContext)' is present in the contract but not in the implementation.
```

Comment:
I don't see why anyone would call this. I guess I made it public just because there is no clear separation between the public API and the internal API.

Reply:
Regardless, this will require a major version change if we break these contracts.

Reply:
I disagree. Some things are public just because they might be useful from outside, they are not part of the API. While I'm not against releasing only major versions, it's not sending the right message, right? :) If people use this stuff, they'll break, but this seems better than not being able to do stuff at all.

Reply:
I think we should simply review what API Compat finds and make a judgement call. In this particular case, it seems simple enough.